python/cugraph/tests/test_modularity.py

```diff
@@ -42,10 +42,10 @@ def random_call(G, partitions):
     return score
 
 
-DATASETS = [
-    '../datasets/karate',
-    '../datasets/dolphins',
-    '../datasets/netscience']
+DATASETS = ['../datasets/karate.csv',
+            '../datasets/dolphins.csv',
+            '../datasets/netscience.csv',
+            '../datasets/email-Eu-core.csv']
```

Comment:
I'd also recommend deleting directed input testing here as it is not supposed to work on undirected input

Reply:
done